### ISSUE TITLE
Solvuu-build installs generic modules globally

### ISSUE THREAD

**Issue by copy** [solvuu/solvuu-build#57]
solvuu-build installs its helper modules (like `Tools`, `Util`), which are already included in the pack. This can cause conflicts with other libraries and triggers [this bug](https://caml.inria.fr/mantis/view.php?id=7349) when using solvuu-build as a library.

```
~ % ls .opam/4.04.0+flambda/lib/solvuu-build/*.cmx
.opam/4.04.0+flambda/lib/solvuu-build/project.cmx               .opam/4.04.0+flambda/lib/solvuu-build/std.cmx
.opam/4.04.0+flambda/lib/solvuu-build/solvuu_build.cmx          .opam/4.04.0+flambda/lib/solvuu-build/tools.cmx
.opam/4.04.0+flambda/lib/solvuu-build/solvuu_build_eliom.cmx    .opam/4.04.0+flambda/lib/solvuu-build/util.cmx
.opam/4.04.0+flambda/lib/solvuu-build/solvuu_build_findlib.cmx

```

**Comment by agarwal** [comment]
I've pushed a new branch `issue-57`, which stops installing `cmi`, `cmo`, `cmx`, and `.o` files. I believe that other files such as `annot`, `cmt` and `cmti` should still be installed, at least that is how Core does it.

If you confirm this solves your problem, I'll merge to master.

**Comment by copy** [comment]
The branch fixes this issue for me, cheers!

**Comment by pveber** [comment]
@agarwal I guess you mean `cmi`, `cmo`, `cmx` and `o` files from packed modules *only*, right?

**Comment by agarwal** [comment]
> I guess you mean cmi, cmo, cmx and o files from packed modules only, right?

Right, yes.

**Comment by pveber** [comment]
Then I agree it should be harmless at worst.

**Comment by agarwal** [comment]
Fixed in 88b92da455e2aa98a1ccc047b767a43c09c60068, which has now been merged to master.
